parser/methods/v8.go

```diff
@@ -0,0 +1,34 @@
+package methods
+
+import (
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/builtin"
+	"github.com/filecoin-project/go-state-types/builtin/v8/account"
+	"github.com/filecoin-project/go-state-types/builtin/v8/cron"
+	filInit "github.com/filecoin-project/go-state-types/builtin/v8/init"
+	"github.com/filecoin-project/go-state-types/builtin/v8/market"
+	"github.com/filecoin-project/go-state-types/builtin/v8/miner"
+	"github.com/filecoin-project/go-state-types/builtin/v8/multisig"
+	"github.com/filecoin-project/go-state-types/builtin/v8/paych"
+	"github.com/filecoin-project/go-state-types/builtin/v8/power"
+	"github.com/filecoin-project/go-state-types/builtin/v8/reward"
+	"github.com/filecoin-project/go-state-types/builtin/v8/verifreg"
+	"github.com/filecoin-project/go-state-types/manifest"
+)
+
+var v8methods = map[string]map[abi.MethodNum]builtin.MethodMeta{
```

Comment:
I'm no sure, but it shouldn't not be necessary to keep track of previous methods, the idea IIRC is that when an onchain upgrade happens, all actors are upgraded old and new ones. So everyone node will be using the 'latest' version of actors since the onchain upgrade takes care of that.

the only case that I'm aware that we need to keep track of older versions is for actors-cids. the only example that I can think of now is that for example an Exec tx that creates a multisig actor: the `params.actorCid` would have the CID value of the multisig-actor that it had at that time, since that doesn't change on upgrades because is inside a block.

But, in short, the support for older seeds lets do it on another PR, and for this one my opinion is that is not necessary tracking v8..v10 since v11 should be backwards compatible

Reply:
I'll get rid of this and left v11 only!